### PR TITLE
Updated some deprecated classes

### PR DESCRIPTION
**Pull request by rzsouza**
And minor tidy up

### CODE REVIEW

**Comment by rzsouza** [comment]
I just saw that some of the files had their EOL characters fixed during the upload. They all had minor changes and it's probably easier to check them in the side by side mode

**Comment by entando-service** [comment]
All is well

File: engine/src/main/java/org/entando/entando/aps/system/services/page/PageTokenManager.java [hunk 1]
```diff
@@ -102,11 +99,9 @@ public String decrypt(String property) {
 			Cipher pbeCipher = Cipher.getInstance("PBEWithMD5AndDES");
 			pbeCipher.init(Cipher.DECRYPT_MODE, key, new PBEParameterSpec(this.getSalt().getBytes(), 20));
 			return new String(pbeCipher.doFinal(base64Decode(property)), "UTF-8");
-		} catch (GeneralSecurityException e) {
-			logger.error("Error in decrypt");
-		} catch (IOException e) {
+		} catch (GeneralSecurityException | IOException e) {
 			logger.error("Error in decrypt");
```
Comment (joewhite101) [inline engine/src/main/java/org/entando/entando/aps/system/services/page/PageTokenManager.java]: If we're changing this we should also log the exception itself `logger.error("Error in decrypt",e);

File: engine/src/main/java/org/entando/entando/aps/system/services/storage/api/LocalStorageManagerInterface.java [hunk 2]
```diff
@@ -40,10 +39,15 @@
 
 public class LocalStorageManagerInterface implements IApiExportable {
 
-	private static final Logger _logger = LoggerFactory.getLogger(LocalStorageManagerInterface.class);
+	private static final Logger logger = LoggerFactory.getLogger(LocalStorageManagerInterface.class);
+
+	private IStorageManager _storageManager;
```
Comment (joewhite101) [inline engine/src/main/java/org/entando/entando/aps/system/services/storage/api/LocalStorageManagerInterface.java]: If touching remove underscore

File: engine/src/main/java/org/entando/entando/aps/system/services/page/PageTokenManager.java (hunk 1 above)
Comment (rzsouza) [reply engine/src/main/java/org/entando/entando/aps/system/services/page/PageTokenManager.java]: Fixed

File: engine/src/main/java/org/entando/entando/aps/system/services/storage/api/LocalStorageManagerInterface.java (hunk 2 above)
Comment (rzsouza) [reply engine/src/main/java/org/entando/entando/aps/system/services/storage/api/LocalStorageManagerInterface.java]: Fixed

**Comment by entando-service** [comment]
All is well

**Pull request merged**